modules/cnf-verify-sriov-operator-n3000.adoc

```diff
@@ -0,0 +1,232 @@
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-bbdev-sample-app
+spec:
+  containers:
+  - securityContext:
+      privileged: false
+      capabilities:
+        add:
+          - IPC_LOCK
+          - SYS_NICE
+    name: bbdev-sample-app
+    image: bbdev-sample-app:1.0
+    command: [ "sudo", "/bin/bash", "-c", "--" ]
+    args: [ "while true; do sleep 300000; done;" ]
+    volumeMounts:
+    - mountPath: /hugepages
+      name: hugepage
+    - name: class
+      mountPath: /sys/devices
+      readOnly: false
+    resources:
+      requests:
+        hugepages-1Gi: 4Gi
+        memory: 1Gi
+        cpu: "4"
+        intel.com/intel_fec_acc100: '1'
+        intel.com/intel_fec_5g: '1'
+        intel.com/intel_fec_lte: '1'
+      limits:
+        memory: 4Gi
+        cpu: "4"
+        hugepages-1Gi: 4Gi
+        intel.com/intel_fec_acc100: '1'
+        intel.com/intel_fec_5g: '1'
+        intel.com/intel_fec_lte: '1'
+  volumes:
+  - name: hugepage
+    emptyDir:
+      medium: HugePages
+  - hostPath:
+      path: "/sys/devices"
+    name: class
+----
+
+. Create the pod:
++
+[source,terminal]
+----
+$  oc  apply -f pod-test.yaml
+----
+
+. Check the pod is created:
++
+[source,terminal]
+----
+$  oc  apply -f pod-test.yaml
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY           STATUS          RESTARTS        AGE
+fec-pod                                         1/1             Running         0               10s
+fpga-driver-daemonset-8xz4c                     1/1             Running         0               15d
+fpgainfo-exporter-vhvdq                         1/1             Running         0               15d
+N3000-controller-manager-b68475c76-gcc6v        2/2             Running         1               15d
+N3000-daemonset-5k55l                           1/1             Running         1               15d
+N3000-discovery-blmjl                           1/1             Running         1               15d
+N3000-discovery-lblh7                           1/1             Running         1               15d
+sriov-device-plugin-j5jlv                       1/1             Running         1               15d
+sriov-fec-controller-manager-85b6b8f4d4-gd2qg   1/1             Running         1               15d
+sriov-fec-daemonset-kqqs6                       1/1             Running         1               15d
+----
+
+. Remote shell login to the fec-pod:
++
+[source,terminal]
+----
+$ oc rsh fec-pod
+----
++
+.Example output
+[source,terminal]
+----
+sh-4.4#
+----
+
+. Print a list of environment variables:
++
+[source,terminal]
+----
+sh-4.4# env
+----
++
+.Example output
+[source,yaml]
+----
+
+N3000_CONTROLLER_MANAGER_METRICS_SERVICE_PORT_8443_TCP_ADDR=172.30.133.131
+SRIOV_FEC_CONTROLLER_MANAGER_METRICS_SERVICE_PORT_8443_TCP_PROTO=tcp
+DPDK_VERSION=28.11
+PCIDEVICE_INTEL_COM_INTEL_FEC_5G=0.0.0.0:1d.00.0 <1>
```

Comment:
add a note this can be of the of three

```
PCIDEVICE_INTEL_COM_INTEL_FEC_ACC100
PCIDEVICE_INTEL_COM_INTEL_FEC_5G
PCIDEVICE_INTEL_COM_INTEL_FEC_LTE
```

depending on the resource the user request in the pod yaml